pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/UnusedPrivateField.xml

```diff
@@ -554,9 +554,11 @@ public class Foo {
     </test-code>
 
     <test-code>
-        <description>#907 UnusedPrivateField false-positive with @FXML</description>
+        <description>#907 UnusedPrivateField false-positive with @FXML - 2</description>
         <expected-problems>0</expected-problems>
         <code><![CDATA[
+        import java.lang.Deprecated;
```

Comment:
Technically, this import should not be needed. Does the test still run without this import?